src/tree/mod.rs

```diff
@@ -293,11 +293,11 @@ impl RatchetTree {
         group_context: &[u8],
         new_leaves_indexes: HashSet<&NodeIndex>,
     ) -> Result<&CommitSecret, TreeError> {
-        let own_index = self.own_node_index();
+        let own_index = NodeIndex::from(self.own_node_index());
```

Comment:
`own_node_index` is not `NodeIndex`? 🤔

Reply:
The own index is a leaf index, which is correct.

Reply:
I think the issue is with the name.